packages/search-engine/src/search-engine.js

```diff
@@ -1,7 +1,39 @@
-'use strict';
+const fastify = require('fastify');
+const mongoose = require('mongoose');
+const dotenv = require('dotenv');
 
 function searchEngine() {
-    // TODO
+  // TODO
 }
 
+dotenv.config();
+// initialized Fastify App
+const app = fastify();
+
+const mongoURI = process.env.CONNECT_DB;
+
+// connected fastify to mongoose
+try {
+  mongoose.connect(mongoURI);
```

Comment:
You should store mongoose instance in some place. In fact I create a way to make a connection to mongodb using mongoose.
First of all you should create a new file to create mongoose connection.
search-engine/src/drivers/mongo/connection.js
To create connection I make a function in shared package.
### Steps to be able to use it

1.  update your branch
2. run `yarn bootstarp` in root directory

### Steps to implement it

1. import MongoDriver in your file (search-engine/src/drivers/mongo/connection.js) `const { MongoDriver } = require('@booking-services/shared');`
2. call the function giving a prefix like a parameter 
example about how my team implement it.

```
const { MongoDriver } = require('@booking-services/shared');
const { prefix } = require('../../utils/constants')

module.exports = MongoDriver.createConnection(prefix);
```

**Note:** createConnection return an instance of mongoose so you can use a variable something like this
`const connection = MongoDriver.createConnection(prefix);
module.exports =  connection;
`